src/System.Runtime/tests/System/Text/StringBuilderTests.cs

```diff
@@ -628,10 +628,13 @@ public static void Append_CharArray_Invalid()
             Assert.Throws<ArgumentNullException>("value", () => builder.Append((char[])null, 1, 1)); // Value is null, startIndex > 0 and count > 0
 
             Assert.Throws<ArgumentOutOfRangeException>("startIndex", () => builder.Append(new char[0], -1, 0)); // Start index < 0
-            Assert.Throws<ArgumentOutOfRangeException>("count", () => builder.Append(new char[0], 0, -1)); // Count < 0
+            Assert.Throws<ArgumentOutOfRangeException>(
+                RuntimeDetection.IsNetFramework ? "count" : "charCount", () => builder.Append(new char[0], 0, -1)); // Count < 0
```

Comment:
Never-mind I think I have followed the full chain of what is going on now. We updated coreclr to use nameof and these parameters didn't match the actual parameter strings. I'm OK with this update (although it makes some of these tests messy).